app/queries/my_facilities/missed_visits_query.rb

```diff
@@ -66,8 +64,8 @@ def calls_made
   def missed_visits_by_facility
     @missed_visits_by_facility ||=
       patients_by_period.map do |(year, period), patients|
-        responsible_patients = patients.group(:bp_facility_id).count
-        visited_patients = visits_by_period[[year, period]].group(:responsible_facility_id).count
+        responsible_patients = patients.group(:registration_facility_id).count
```

Comment:
Should we rename this to `registered_patients`?

Reply:
I like keeping it as `responsible_patients`. This solidifies the idea that we will eternally care about responsible patients, and our definition of what "responsible" means may shift over time.